### ISSUE TITLE
Compress map files

### ISSUE THREAD

**Issue by moonheart08** [griefly/griefly#487]
Pretty simple, map files should *probably* be compressed. At the moment maps are gigantic.

**Comment by kremius** [comment]
Indeed, it would be wise to allow map be compressed. It will be helpful for game distribution.
However, for _development_ it is more practical to have map files as pure json.
1. Easier to debug.
2. Git diffs should behave much better with json than with zip, because compressed files change too randomly.

So it should be optional feature (i.e. the game should support both compressed/uncompressed map files).
